stac_ipyleaflet/core.py

```diff
@@ -78,47 +84,30 @@ def __init__(self, **kwargs):
         self.loading_widget_layer = Popup(
             child=loading_widget, min_width=200, min_height=200
         )
-
-        main_button_layout = Layout(
-            width="120px", height="35px", border="1px solid #4682B4"
+        return
+
+    def create_buttons_layout(self):
+        interact_btn = self.create_widget_button(
+            "interact",
+            "Interact with the map",
+            "Interact",
+            "pencil",
+            self.toggle_interact_widget_display,
         )
-        # @NOTE: Break these button creations out...
-        interact_btn = ToggleButton(
-            description="Interact", icon="pencil", layout=main_button_layout
+        layers_btn = self.create_widget_button(
+            "layers",
+            "Open/Close Layers Menu",
+            "Layers",
+            "map-o",
+            self.toggle_layers_widget_display,
```

Comment:
Is it possible to still keep the param names, while creating these buttons in a custom function? It helps with readibility - saves from going to the `create_widget_button` function itself.

Reply:
spoke to @emmalu here, can add class type for type hinting purposes